ledger-core/v2/virtual/stateexchange/vstatereport.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
Move file to `virtual/request/...`, where all handlers are

Reply:
@bigbes may be we rename that package to `handlers` in a separate PR?